OpenUtau.Core/Enunu/EnunuConfig.cs

```diff
@@ -1,5 +1,6 @@
 ﻿using System.Collections.Generic;
 using System.IO;
+using System.Security.Cryptography.X509Certificates;
```

Comment:
You may have include System.Security.Cryptography.X509Certificates by mistake. visual studio often includes this when coding even if we don't need it.